docs/src/integrations/source/troubleshooting.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Troubleshooting"
+description: |
+  In the following section you'll find troubleshooting guidance on source integration.
+---
+
+{{< description >}}
+
+## Troubleshooting while cloning the repository
+
+When you set up an [external integration](https://docs.platform.sh/integrations/source.html) to GitHub, GitLab, or Bitbucket, an additional layer of access control gets added.
```

Comment:
As I mentioned in the other PR, this isn't the [proper format for links](https://github.com/platformsh/platformsh-docs/blob/master/contributing/markup-format.md#links).

Also, it's not clear to me how this relates to troubleshooting. This seems like a general statement that belongs on the overview page. Or maybe just more context.

Reply:
This was the original text. Maybe it could go up in the description?

Reply:
I wouldn't say it's the description of the page as a whole, but it could work as an explanation why the problem might arise :+1: